docs/core/misc/slip0039.md

```diff
@@ -0,0 +1,107 @@
+that any passphrase is valid, and using any passphrase will yield a working wallet.
+
+SLIP-39 enables this usage by specifying that passphrases are not validated in any way.
+Decrypting an EMS with any passphrase will produce data usable as the Master Secret,
+regardless of whether it is the original data or not.
+
+
+## Seed handling in Trezor
+
+Trezor stores a _mnemonic secret_ in a storage field `_MNEMONIC_SECRET`. This is the
+input for the root node derivation process: `mnemonic.get_seed(passphrase)` takes the
+user-provided passphrase as an argument, and derives the appropriate root node from the
+mnemonic secret.
+
+With BIP-39, the recovery phrase itself is the mnemonic secret. During device
+initialization, the raw recovery phrase is given to the user, and also directly stored
+in the `_MNEMONIC_SECRET` field. Whenever the root node is required, it is derived by
+applying PBKDF2 to the mnemonic secret plus passphrase.
+
+For SLIP-39 it is not practical to store the raw data of the recovery shares. Instead,
+the mnemonic secret is a randomly-generated 128 or 256 bit token. SLIP-39 encryption
+parameters (a random identifier and an iteration exponent) are stored alongside the
+mnemonic secret in their own storage fields.
+
+The mnemonic secret is then treated as an Encrypted Master Secret. Every time the root
+node is required, it is derived by "decrypting" the mnemonic secret with the provided
+passphrase.
+
+
+## SLIP-39 implementation
+
+The [reference implementation](https://github.com/trezor/python-shamir-mnemonic) of
+SLIP-39 provides the following high-level API:
+
+* `generate_mnemonics(group parameters, master_secret, passphrase)`: Encrypt Master
+   Secret with the provided passphrase, and split into a number of shares defined via
+   the group parameters. 
+   Implemented using the following:
+   - `encrypt(master_secret, passphrase, iteration_exponent, identifier)`: Encrypt the
+     Master Secret with the given passphrase and parameters.
+   - **`split_ems(group parameters, identifier, iteration_exponent, encrypted_master_secret)`**:
+     Split the encrypted secret and encode the metadata into a set of shares defined via
+     the group parameters.
+* `combine_mnemonics(set of shares, passphrase)`: Combine the given set of shares to
+  reconstruct the secret, then decrypt it with the provided passphrase. 
+  Implemented using the following:
+  - **`recover_ems(set of shares)`**: Combine the given set of shares to obtain the
+    encrypted master secret, identifier and iteration exponent.
+  - **`decrypt(encrypted_master_secret, passphrase, iteration_exponent, identifier)`**:
+    Decrypt the secret with the given passphrase and parameters, to obtain the original
+    Master Secret.
+
+Only the functions denoted in **bold** are implemented in trezor-core. Splitting of the
+backup is done with `split_ems/recover_ems`, and passphrase decryption is done with
```

Comment:
Splitting of the backup is done with `split_ems` only.

Reply:
"splitting and recovering the backup" then?

Reply:
Could be. I would go for "Backup and recovery are done using ..." or "Recovery shares are generated using `split_ems` and combined using `recover_ems`"